ydb/docs/en/core/reference/docker/environment.md

```diff
@@ -0,0 +1,31 @@
+| [`GRPC_TLS_PORT`](https://GitHub.com/ydb-platform/ydb/blob/8dde59cd0af86737d07a1cd8ff19811a2bd2b663/ydb/tests/library/harness/kikimr_port_allocator.py#L183)                | `int`      | `2137`                 |                              | `GPRC` port for secure connection                                                                                                                                                      |
+| [`YDB_ERASURE`](https://GitHub.com/ydb-platform/ydb/blob/8fefc809c83829d8d8b886e82534d009de4c8826/ydb/public/tools/lib/cmds/__init__.py#L50)                                | `string`   | `None`                 |                              | Erasure Coding method, see. [{#T}](../../concepts/topology.md)                                                                                                                         |
+| [`FQ_CONNECTOR_ENDPOINT`](https://GitHub.com/ydb-platform/ydb/blob/c113fcffa7b1a20ad8dcb1b1760ae5bfa25370ca/ydb/public/tools/lib/cmds/__init__.py#L261)                     | `string`   | `None`                 | `≥24.1`                      | Connection string for the connector to external sources `fq-connector-go`                                                                                                              |
+| [`YDB_USE_IN_MEMORY_PDISKS`](https://GitHub.com/ydb-platform/ydb/blob/c113fcffa7b1a20ad8dcb1b1760ae5bfa25370ca/ydb/public/tools/lib/cmds/__init__.py#L230)                  | `0` or `1` | `0`                    |                              | Makes all data volatile and stored only in RAM. Currently, saving data by disabling it is supported only on x86_64 processors                                                          |
+| [`YDB_DEFAULT_LOG_LEVEL`](https://GitHub.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L73)                 | `string`   | `NOTICE`               |                              | Sets the logging level by default. Available level values: `CRIT`, `ERROR`, `WARN`, `NOTICE`, `INFO`.                                                                                  |
+| [`YDB_ADDITIONAL_LOG_CONFIGS`](https://GitHub.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L48)            | `string`   | `None`                 |                              | Sets additional logging levels in format: `level`:`level value`. If you need to enter multiple levels, then enter them separated by commas.                                            |
```

Comment:
It's unclear what the difference is between "level" and "level value". Is this about per-component logging level changes? If so, it needs a link to the list of available components too.

An example won't hurt either, as this is a less trivial option compared to neighboring ones.